src/data/corpus.py

```diff
@@ -0,0 +1,46 @@
+import re
+
+document = str(input("Please insert the full path to the text (without quotation marks): "))
+
+f = open(document, "r", encoding="utf-8")
+text = f.read()
+
+
+def un_gutenberg(text):
+    """Removes the Project Gutenberg intro and outro"""
+    new_text = re.search("[*][*][*] [a-zA-ZÀ-ú.:;,!-? ]+ [*][*][*]", text)
+    no_intro = text[new_text.end():].strip()
+
+    new_text = re.search("[*][*][*] [a-zA-ZÀ-ú.:;,!-? ]+ [*][*][*]", no_intro)
+    no_outro = no_intro[:new_text.start()].strip()
+
+    return no_outro
+
+
+beginnings = ["Chapitre I\n", "CHAPITRE I\n", "I\n", "LIVRE PREMIER\n", "Livre premier\n"]
+endings = ["NOTES", "NOTE", "Notes"]
+
+
+def clean_up(text):
+    """Tries to clean up the text, removing the beginning notes,
```

Comment:
in doctrings, no need to write "tries" or so, just write verbs in imperative form to express the  idea:
in this case: "clean up the text... etc etc."